README.md

```diff
@@ -1036,6 +1036,7 @@ Table of Contents
    * [plan.io](https://plan.io/) — Project Management with Repository Hosting and more options. Free for 2 users with 10 customers and 500MB Storage
    * [planitpoker.com](https://www.planitpoker.com/) — Free online planning poker (estimation tool)
    * [saas.zentao.pm](https://saas.zentao.pm/) - An Application Lifecycle Management solution for Issue Tracking and Project Management, on-premise and open source version are available as well.
+   * [ScrumFast](https://www.scrumfast.com) - Free Scrum board with a very intuitive interface.
```

Comment:
please rather say its free for 5 users

Reply:
@ripienaar Updated as requested. Thanks!